gradle/scripts/lib/java.gradle

```diff
@@ -92,7 +92,21 @@ configure(projectsWithFlags('java')) {
         testLogging.exceptionFormat = 'full'
 
         // Use JUnit platform.
-        useJUnitPlatform()
+        def flakyTest = rootProject.findProperty('flakyTest')
+        if (flakyTest == 'true') {
+            useJUnitPlatform {
+                includeTags 'FLAKY_TEST'
+            }
+        } else if (flakyTest == 'false') {
+            useJUnitPlatform {
+                excludeTags 'FLAKY_TEST'
```

Comment:
Should we use a plural form, e.g. `flakyTests` and `FLAKY_TESTS`?

Reply:
Initially, I used a singular form for consistency with `@FlakyTest` annotation, but a plural form sounds good.